iron-ajax.html

```diff
@@ -353,13 +353,15 @@
         contentType = 'application/x-www-form-urlencoded';
       }
       if (contentType) {
-        headers['Content-Type'] = contentType;
+        headers['content-type'] = contentType;
       }
       var header;
 
       if (this.headers instanceof Object) {
         for (header in this.headers) {
-          headers[header] = this.headers[header].toString();
+          // Normalize headers in lower case to make it easier for iron-request
+          // to reason about them.
+          headers[header.toLowerCase()] = this.headers[header].toString();
```

Comment:
Header names are specified to be case insensitive, and AFAIK most modern web servers are on board with this. That said, this kind of arbitrarily screws anyone who happens to be forced to deal with a legacy backend that cares about the case of headers.

I'm not convinced that's a great reason to make our implementation more complex. But, WDYT?